File: examples/nodejs/simple/example_1.js
const MdisClient = require("./src/index");

async function main() {
  try {
    let m = MdisClient.connect((host = "127.0.0.1"), (port = 6411));
    const resp = await m.set("token", "123456");
    console.log(resp);
    // console.log("completed");
  } catch (error) {
    console.log(error);
  }
}
main();

File: examples/c/simple/example_1.c
/* Example 1: SET a value. */
#include <stdio.h>
#include <stdlib.h>

#include "src/mdis_client.h"

int main(void) {
    MdisClient client = mdis_connect("127.0.0.1", 6411);

    char *resp = mdis_set(&client, "token", "123456", 0);
    if (!resp) {
        fprintf(stderr, "Error: could not reach server\n");
        return 1;
    }

    printf("response: %s\n", resp);
    free(resp);
    return 0;
}
